cmd/syft/cli/options/writer.go

```diff
@@ -55,6 +56,7 @@ func parseOutputs(outputs []string, defaultFile string) (out []sbom.WriterOption
 			errs = multierror.Append(errs, fmt.Errorf("bad output format: '%s'", name))
 			continue
 		}
+		format = format.WithOptions(options.Format{TemplateFilePath: templateFilePath})
```

Comment:
another way to achieve this without changing the Format interface is to type assert to `template.Format`, and if the assertion is ok, then add on the additional information or create a new instance of the format with the new information injected via a constructor.

Reply:
yes, I like that idea. I used and that helped drop `options.Format`.